gem/lib/pupperware/spec_helper.rb

```diff
@@ -364,6 +364,66 @@ def wait_for_pe_orchestration_services()
     end
   end
 
+  ######################################################################
+  # PE Bolt Server Helpers
+  ######################################################################
+
+  def curl_job_number(job_number)
+    #Wait for 30 seconds for the task to run
+    timeout = 30
+    puts "Waiting for the task to run..."
+    return retry_block_up_to_timeout(timeout) do
+      output = curl('localhost', 443, "api/jobs/#{job_number}").body
+      puts output
+      output !~ /running/ ? output :
+        raise("Job was still running after #{timeout} seconds")
+    end
+    return output
+  end
+
+  def curl_console_task()
+    uri = URI.parse("https://localhost:443/api/tasks/create")
+    request = Net::HTTP::Post.new(uri)
+    request.content_type = "application/json"
+    request["X-Authentication"] = @rbac_token
+    request.body = JSON.dump({
+                               "nodes" => [
+                                 "test_sshd.test"
```

Comment:
Should `nodes` and `hostnames` below be parametrizable in this method?

Reply:
Good call on these changes!